### PR TITLE
Use a PAT for hopefully less Github authentication issues.

### PR DESCRIPTION
**Pull request by lonnieezell**

### CODE REVIEW

File: .github/workflows/deptrac.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
           extensions: intl, json, mbstring, xml
           coverage: none
         env:
-          COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPOSER_TOKEN: ${{ secrets. }}
```
Comment (kenjis) [inline .github/workflows/deptrac.yml:39]: Missing variable name.

Comment (lonnieezell) [reply .github/workflows/deptrac.yml]: Good catch. Thanks. Fixed.

**Review by kenjis — APPROVED** [review]

**Review by MGatner — COMMENTED** [review]
Note that `GITHUB_TOKEN` is the recommendation here (https://github.com/shivammathur/setup-php). I think you may have reintroduced the auth issues with this.

**Comment by lonnieezell** [comment]
Hm, well everywhere I look that talks about issues with authentication using GITHUB_TOKEN says to use a PAT if you're having problems. But this obviously didn't help out like we'd hoped. Damn. There's got to be a way to fix this. As it stands there's no way to reliably know if a PR messes something up or not.

**Comment by MGatner** [comment]
Yeah, I'm not saying I know for sure as this has been going on across many different workflow iterations across many different repos - this is just what `ssteup-php` recommends (more precise link: https://github.com/shivammathur/setup-php#composer-github-oauth).

**Pull request merged**